tools.repos

```diff
@@ -3,3 +3,7 @@ repositories:
     type: git
     url: https://github.com/autowarefoundation/autoware_tools.git
     version: 0.2.0
+  plotjuggler_ros:
```

Comment:
Could you add a comment saying that we can remove this repo again once https://github.com/PlotJuggler/plotjuggler-ros-plugins/pull/94 is merged and the binary package is available again?

Reply:
Done.
https://github.com/autowarefoundation/autoware/pull/6153/commits/1f1b785c14b8fafe92e44cdacc037e74320f0823